.bumpversion.cfg

```diff
@@ -4,3 +4,11 @@ commit = False
 tag = False
 
 [bumpversion:file:src/server/_config.py]
+
+[bumpversion:file:src/client/delphi_epidata.js]
+
+[bumpversion:file:src/client/delphi_epidata.R]
+
+[bumpversion:file:src/client/packaging/npm/package.json]
+
+[bumpversion:file:src/client/packaging/pypi/setup.py]
```

Comment:
do you also need

```
[bumpversion:file:src/client/packaging/pypi/delphi_epidata/__init__.py]
```

?

Reply:
good catch, I was looking at the delphi_epidata.py only which is extracting the version